src/widget/wlibrary.cpp

```diff
@@ -142,3 +142,9 @@ bool WLibrary::event(QEvent* pEvent) {
     }
     return QStackedWidget::event(pEvent);
 }
+
+void WLibrary::keyPressEvent(QKeyEvent* event) {
+    if (event->key() == Qt::Key_Left && event->modifiers() & Qt::ControlModifier) {
+        emit setLibraryFocus(FocusWidget::Sidebar);
+    }
```

Comment:
Do we need to chain call the base class implementation here?

Reply:
I guess it's cleaner to do so, but I don't think QStackedWidget does any key handling. Anyway, I will make the change.

Reply:
Done